model/host/db.go

```diff
@@ -428,7 +429,7 @@ func RemoveStaleInitializing(distroID string) error {
 			},
 			{
 				StatusKey:     evergreen.HostBuilding,
-				CreateTimeKey: bson.M{"$lt": time.Now().Add(-15 * time.Minute)},
+				CreateTimeKey: bson.M{"$lt": time.Now().Add(-20 * time.Minute)},
```

Comment:
Where did the 15 -> 20 minute change come from?

Reply:
This was because it looked like it was taking ~19 minutes for the entire process of importing, building, and creating the container to complete. However, looking at the Splunk logs, it looks like it has gone back to ~12 minutes. I will change it back to 15 now.